### PR TITLE
Update Trilinos and Kokkos-tools in Docker

### PR DESCRIPTION
**Pull request by aprokop**
1. Updating Trilinos is required to merge #554.
2. Updating Kokkos tools is optional
3. Replacing NetCDF url as am unable to download from the FTP server.
   Tested locally with `data` that the Exodus test passes.

@dalg24 Can you please update and upload containers with the new Dockerfile?

### CODE REVIEW

**Comment by dalg24** [comment]
> @dalg24 Can you please update and upload containers with the new Dockerfile?

You are a collaborator on the registry.

**Comment by aprokop** [comment]
> You are a collaborator on the registry.

Oh, OK, did not know that. I'm trying to look recipes for the exact docker images I need to build. I thought I saw them somewhere, but have trouble finding.

**Comment by dalg24** [comment]
See [docker/README.md](https://github.com/ORNL-CEES/DataTransferKit/blob/4fbba5de3b77e3c3c70bddfd92e7fda5707bdfa8/docker/README.md#19010-january-14th-2019), new tag is `19.06.0`

**Comment by aprokop** [comment]
retest this please

**Comment by aprokop** [comment]
retest this please

**Comment by dalg24** [comment]
Did you change the value of the `GPU_ARCH` environment variable?

**Comment by aprokop** [comment]
> Did you change the value of the `GPU_ARCH` environment variable?

No.

**Comment by aprokop** [comment]
retest this please

**Comment by aprokop** [comment]
retest this please

**Comment by aprokop** [comment]
retest this please

**Comment by aprokop** [comment]
retest this please

**Comment by aprokop** [comment]
YAY!!! 

@dalg24 Keep your fingers away from this PR.

**Comment by dalg24** [comment]
r3t3st dis ple4z

File: docker/CHANGELOG.md [hunk 1]
```diff
@@ -1,45 +1,11 @@
...
-COMMENT: downloaded binaries for Clang + LLVM (including support for OpenMP) instead of using system package manager/building from source
-- Upgrade Open MPI to 2.1.3
-
-## 18.09.0 (September 5th, 2018)
-### TPLs addition
-- Add benchmark support library from google (version 1.4.1)
+- Updated Kokkos Profiling to 2.7.24
+- Updated Trilinos from a27130b to 2a24058e (merge of Kokkos 2.8.0)
+- Updated netcdf repo location
+  NOTE: netcdf's ftp server is unaccessible
```
Comment (dalg24) [inline docker/CHANGELOG.md]: It is spelled NetCDF

Comment (aprokop) [reply docker/CHANGELOG.md]: Won't fix.

**Comment by aprokop** [comment]
This PR should be merged after #554 is tested clean.

File: docker/CHANGELOG.md (hunk 1 above)
Comment (Rombur) [reply docker/CHANGELOG.md]: but it is not consistent with Line 43

**Pull request merged**